src/commands/chain.js

```diff
@@ -82,6 +82,19 @@ async function main(archwayd, name, options = {}) {
                 process.stdout.write(`${data}`);
             });	
             break;
+        case 'snapshot': 
+            const today = new Date();
+            console.log("Make a snapshot");
+            const mkDir = spawnSync("mkdir", [`${archwayd.archwaydHome}/snapshot`]);
+            if (mkDir.error != null){
+                throw mkDir.error;
+            }
+            const snapshot = spawnSync("tar", ["zcvf", `${archwayd.archwaydHome}/snapshot/${today.getFullYear()}${today.getMonth()}${today.getDate()}_data.tar.gz`,`${archwayd.archwaydHome}/data`]);
```

Comment:
As I remember, we discuss that store new snapshot into `archway` project directory.
But as I see here created snapshot stores into chain's home.

Would you keep this impl. or change that?
I think there's no problem whatever if you're ok.